CMakeLists.txt

```
@@ -59,30 +59,36 @@ if (UPDATE_DEPS)
         set(optional_args ${optional_args} "CMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
     endif()
 
+    set(_update_deps_dir ${CMAKE_CURRENT_LIST_DIR}/external-${_build_type})
```

Keep all installed dependencies in 1 folder.

```suggestion
    set(_update_deps_dir ${CMAKE_CURRENT_LIST_DIR}/external/${_build_type})
```

---

Also removes the need to update the `.gitignore`

---

Yep, much better, thanks!